apps/astarte_housekeeping_api/lib/astarte_housekeeping_api/realms/device_registration_limit_type.ex

```diff
@@ -0,0 +1,38 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2023 SECO Mind srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.Housekeeping.API.Realms.DeviceRegistrationLimitType do
+  use Ecto.Type
+  def type, do: :any
+
+  def cast(:remove), do: {:ok, :remove}
```

Comment:
I'd use `:remove_limit` as atom here, as you do in other places. This makes it more readable because it's consistent.